Library/Homebrew/cmd/doctor.rb

```diff
@@ -424,10 +424,18 @@ def check_access_share_man
     __check_subdir_access "share/man"
   end
 
+  def check_access_homebrew_repository
+    unless HOMEBREW_REPOSITORY.writable_real? then <<-EOS.undent
+      The #{HOMEBREW_REPOSITORY} is not writable.
+      You should probably `chown` #{HOMEBREW_REPOSITORY}
```

Comment:
Alright. I guess I'll try and be trusting that if a `chown` fails users will know to try again with `sudo`.